src/code_10400_len_d30.c

```diff
@@ -14,9 +56,31 @@ void state_drawUI_language_select(void) {
 void state_drawUI_file_select(void) {
 }
 
-INCLUDE_ASM(s32, "code_10400_len_d30", func_800355D4);
+void func_800355D4(void) {
+    if (D_800A0932[0] > 0) {
+        D_800A0931 = 0;
+        set_transition_stencil_zoom_0(0, D_800A0932[0]);
```

Comment:
can the `set_transition_stencil_zoom_0(0, D_800A0932[0]);` line be moved after the if/else? It seems redundant to have it be in both cases exactly the same

Reply:
sticking it after the if/else changes the asm quite a bit